juha/programmers/3/자릿수-더하기.js

```diff
@@ -0,0 +1,15 @@
+function solution(n) {
+  let answer = String(n)
+    .split("")
+    .reduce((accumulator, current) => accumulator + parseInt(current), 0);
```

Comment:
string 형태의 숫자를 number 형태로 쉽게 변환하는 방법에는 Number('숫자') 도 있습니다.
```suggestion
    .reduce((accumulator, current) => accumulator + Number(current), 0);
```